### PR TITLE
system_output: set verbose to False in some functions

### PR DESCRIPTION
**Pull request by hereischen**
Set verbose to False to avoid too many lines
of output in debug.log

Signed-off-by: Haotong Chen <hachen@redhat.com>

id: 1555175

### CODE REVIEW

**Comment by hsmj1412** [comment]
I'm very sorry to have troubled your work
https://github.com/avocado-framework/avocado-vt/pull/1417

**Comment by hereischen** [comment]
@hsmj1412  No worries.

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -1371,9 +1371,11 @@ def sort_key(dev):
 
         self.qemu_binary = qemu_binary
         self.qemu_version = process.system_output("%s -version" % qemu_binary,
+                                                  verbose=False,
```
Comment (CongLi) [inline virttest/qemu_vm.py]: why disable version info?

Comment (hereischen) [reply virttest/qemu_vm.py]: We have it at the preprocess phase.
This one's output is 
06:09:47 INFO | Running '/sbin/ovs-vswitchd --version'
06:09:47 DEBUG| [stdout] ovs-vswitchd (Open vSwitch) 2.0.0
06:09:47 INFO | Command '/sbin/ovs-vswitchd --version' finished with 0 after 0.0038321018219s
06:09:47 DEBUG| [stdout] Compiled Jan 26 2014 14:22:04
06:09:47 DEBUG| [stdout] OpenFlow versions 0x1:0x1
06:09:47 INFO | Running '/sbin/ovs-vswitchd --version'
06:09:47 DEBUG| [stdout] ovs-vswitchd (Open vSwitch) 2.0.0
06:09:47 INFO | Command '/sbin/ovs-vswitchd --version' finished with 0 after 0.0037670135498s
06:09:47 DEBUG| [stdout] Compiled Jan 26 2014 14:22:04

**Comment by CongLi** [comment]
1. could you confirm which is disabled in add_pci_controllers()?

**Comment by YongxueHong** [comment]
(workspace) [root@ibm-p8-rhevm-08 staf-kvm-devel]# python ConfigTest.py --testcase=boot --driveformat=virtio_scsi --imageformat=qcow2 --nicmodel=virtio_net --guestname=RHEL.7.5 --platform=ppc64le --mem=8192 --vcpu=8 --verbose=no

Running test by avocado
=> avocado run --journal --vt-config /home/automation2/staf-kvm-devel/workspace/var/lib/avocado/data/avocado-vt/backends/qemu/cfg/tests-example.cfg
JOB ID     : 87278b30d4b99b9c5ff9fa89696493d8a4f39c42
JOB LOG    : /home/automation2/staf-kvm-devel/workspace/job-results/job-2018-03-14T02.53-87278b3/job.log
 (1/3) smp_4.8192m.repeat1.pre_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (1448.11 s)
 (2/3) smp_4.8192m.repeat1.pre_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.rh_kernel_update: PASS (135.65 s)
 (3/3) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.boot: PASS (56.88 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1644.68 s
JOB HTML   : /home/automation2/staf-kvm-devel/workspace/job-results/job-2018-03-14T02.53-87278b3/results.html

Part of debug.log:
...
03:19:42 INFO | Created qemu process with parent PID 6375
03:19:42 INFO | Connecting to monitor '<virttest.qemu_monitor.QMPMonitor> qmpmonitor1'
03:19:42 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'qmp_capabilities' 
03:19:42 DEBUG| Send command: {'execute': 'qmp_capabilities', 'id': 'pKNSrUyP'}
03:19:42 INFO | Connecting to monitor '<virttest.qemu_monitor.QMPMonitor> catch_monitor'
03:19:43 DEBUG| (monitor avocado-vt-vm1.catch_monitor) Sending command 'qmp_capabilities' 
03:19:43 DEBUG| Send command: {'execute': 'qmp_capabilities', 'id': 'YM45Ktdd'}
03:19:43 DEBUG| VM appears to be alive with PID 6377
03:19:43 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'query-cpus' 
03:19:43 DEBUG| Send command: {'execute': 'query-cpus', 'id': 'sfA3GhXt'}
...
03:19:43 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'cont' 
03:19:43 DEBUG| Send command: {'execute': 'cont', 'id': 'DFmQxomB'}
03:19:43 INFO | Context: Try to log into guest 'avocado-vt-vm1'.
03:19:43 DEBUG| Attempting to log into 'avocado-vt-vm1' (timeout 240s)
03:19:43 INFO | Running '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136'
03:20:04 DEBUG| [stdout] ARPING 10.16.70.136 from 10.16.69.65 switch
03:20:04 DEBUG| [stdout] Sent 21 probes (21 broadcast(s))
03:20:04 DEBUG| [stdout] Received 0 response(s)
03:20:04 INFO | Command '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136' finished with 1 after 21.0377910137s
03:20:04 DEBUG| Dropped the address cache of HWADDR (9a:c4:c5:c6:c7:c8)
03:20:08 DEBUG| Updated HWADDR (9a:c4:c5:c6:c7:c8)<->(10.16.70.136) IP pair into address cache
03:20:08 INFO | Running '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136'
03:20:08 DEBUG| [stdout] ARPING 10.16.70.136 from 10.16.69.65 switch
03:20:08 DEBUG| [stdout] Unicast reply from 10.16.70.136 [9A:C4:C5:C6:C7:C8]  1.388ms
03:20:08 DEBUG| [stdout] Sent 1 probes (1 broadcast(s))
03:20:08 DEBUG| [stdout] Received 1 response(s)
03:20:08 INFO | Command '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136' finished with 0 after 0.0450060367584s
03:20:08 DEBUG| Found/Verified IP 10.16.70.136 for VM avocado-vt-vm1 NIC 0
03:20:08 INFO | Running '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136'
03:20:08 DEBUG| [stdout] ARPING 10.16.70.136 from 10.16.69.65 switch
03:20:08 DEBUG| [stdout] Unicast reply from 10.16.70.136 [9A:C4:C5:C6:C7:C8]  2.141ms
03:20:08 DEBUG| [stdout] Sent 1 probes (1 broadcast(s))
03:20:08 DEBUG| [stdout] Received 1 response(s)
03:20:08 INFO | Command '/sbin/arping -f -c3 -w20 -I switch 10.16.70.136' finished with 0 after 0.0694780349731s
03:20:10 DEBUG| Destroying VM avocado-vt-vm1 (PID 6377)
03:20:10 DEBUG| Shutting down VM avocado-vt-vm1 (shell)
03:20:20 INFO | [qemu output] (Process terminated with status 0)
03:20:20 DEBUG| VM avocado-vt-vm1 down (shell)
03:20:20 DEBUG| Host does not support OpenVSwitch: Command 'ovs-vswitchd' could not be found in any of the PATH dirs: ['/usr/lib64/qt-3.3/bin', '/bin', '/sbin', '/usr/bin', '/usr/local/sbin', '/home/automation2/staf-kvm-devel/workspace/bin', '/usr/libexec', '/root/bin', '/usr/sbin', '/usr/local/bin']
03:20:20 DEBUG| Could not find params qemu-img in /home/automation2/staf-kvm-devel/workspace/avocado-vt/backends/qemu/qemu-img, searching the host PATH for one to use
03:20:20 DEBUG| Found /bin/qemu-img
03:20:20 DEBUG| Checking image file /home/automation2/staf-kvm-devel/vt_test_images/rhel75-ppc64le-virtio-scsi.qcow2
03:20:20 DEBUG| Run qemu-img info comamnd on /home/automation2/staf-kvm-devel/vt_test_images/rhel75-ppc64le-virtio-scsi.qcow2
03:20:20 INFO | Running '/bin/qemu-img info -U /home/automation2/staf-kvm-devel/vt_test_images/rhel75-ppc64le-virtio-scsi.qcow2'
03:20:20 DEBUG| [stdout] image: /home/automation2/staf-kvm-devel/vt_test_images/rhel75-ppc64le-virtio-scsi.qcow2
03:20:20 INFO | Command '/bin/qemu-img info -U /home/automation2/staf-kvm-devel/vt_test_images/rhel75-ppc64le-virtio-scsi.qcow2' finished with 0 after 0.0192530155182s
03:20:20 DEBUG| [stdout] file format: qcow2
03:20:20 DEBUG| [stdout] virtual size: 20G (21474836480 bytes)
03:20:20 DEBUG| [stdout] disk size: 4.3G
03:20:20 DEBUG| [stdout] cluster_size: 65536
03:20:20 DEBUG| [stdout] Format specific information:
03:20:20 DEBUG| [stdout]     compat: 1.1
03:20:20 DEBUG| [stdout]     lazy refcounts: false
03:20:20 DEBUG| [stdout]     refcount bits: 16
03:20:20 DEBUG| [stdout]     corrupt: false
03:20:20 DEBUG| Encoding video file /home/automation2/staf-kvm-devel/workspace/job-results/job-2018-03-14T02.53-87278b3/test-results/3-smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.boot/screendumps_avocado-vt-vm1_6377_iter0.webm
03:20:23 WARNI| avocado-vt-vm1 is not alive. Can't query the register status
03:20:34 DEBUG| Thread quit. Used to failed to get register info from guest 20180314-031940-rYJGMEC2 for 4 times.
03:20:35 WARNI| Libvirtd service is not available in host, utils_libvirtd module will not function normally
03:20:35 INFO | Running 'true'
03:20:35 INFO | Command 'true' finished with 0 after 0.000743865966797s
03:20:35 INFO | Running 'ps -o comm 1'
03:20:35 DEBUG| [stdout] COMMAND
03:20:35 INFO | Command 'ps -o comm 1' finished with 0 after 0.01198387146s
03:20:35 DEBUG| [stdout] systemd
03:20:35 DEBUG| DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
03:20:35 DEBUG| DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
03:20:35 DEBUG| DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
03:20:35 INFO | PASS 3-smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.boot

**Comment by YongxueHong** [comment]
Acked-by: Yongxue Hong <yhong@redhat.com>

File: virttest/utils_net.py [hunk 2]
```diff
@@ -3105,7 +3105,7 @@ def __arping(ip, macs, dev, timeout):
         arping_cmd = "%s -f -c3 -w%d -I %s %s" % (arping_bin, int(timeout),
                                                   dev, ip)
         try:
-            o = process.system_output(arping_cmd, shell=True)
```
Comment (chunfuwen) [inline virttest/utils_net.py]: For arping output, we may retain verbose ouput to debug purpose

Comment (hereischen) [reply virttest/utils_net.py]: I would think if it is for the debug purpose, one should enable it when running the case that needs to be debugged. 
Not all cases.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**